src/components/Board/GooglePhotosSearch/GooglePhotosButton/GooglePhotosConnect.Button.js

```diff
@@ -0,0 +1,18 @@
+/*this button is created using Google Photos UXguidelines https://developers.google.com/photos/library/guides/ux-guidelines*/
+import React from 'react';
+import './GooglePhotosConnect.Button.css';
+import { injectIntl } from 'react-intl';
+import messages from './../GooglePhotosSearch.messages';
+
+export default injectIntl(function ConnectToGooglePhotosButton(props) {
+  return (
+    <div className="customBtn">
+      <span className="icon" />
+      <span className="buttonText">
+        {props.intl.formatMessage(messages.addFrom)}
```

Comment:
remove "Add from " from title, that should be a message in the tooltip.

Reply:
i think that we can't use the 'Google Photos' brand alone. https://developers.google.com/photos/library/guides/ux-guidelines
for that, I created a new special button. what do you think?